python/cloudtik/runtime/spark/scripts/configure.sh

```diff
@@ -390,13 +390,11 @@ function configure_jupyter_for_spark() {
 }
 
 function configure_local_hdfs_fs() {
-    # Nothing to do now
-    :
+    sudo sed -i 's?#user_allow_other?user_allow_other?' /etc/fuse.conf
```

Comment:
The code assumes fuse.conf has "#user_allow_other". But this might be quite dynamic depending future fuse installations. When there is no #user_allow_other, we may still need the user_allow_other flag in fuse.conf?